public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -89,17 +90,18 @@ function ViewAction({ permissions, alertManagerName, receiverName }: ActionProps
 }
 
 function ExportAction({ permissions, receiverName }: ActionProps) {
+  const canReadSecrets = contextSrv.hasAccess(permissions.provisioning.readSecrets, isOrgAdmin());
   return (
-    <Authorize actions={[permissions.provisioning.read]} fallback={isOrgAdmin()}>
+    <Authorize actions={[permissions.provisioning.read, permissions.provisioning.readSecrets]}>
       <ActionIcon
         data-testid="export"
         to={createUrl(`/api/v1/provisioning/contact-points/export/`, {
           download: 'true',
           format: 'yaml',
-          decrypt: isOrgAdmin().toString(),
+          decrypt: canReadSecrets.toString(),
           name: receiverName,
         })}
-        tooltip={isOrgAdmin() ? 'Export contact point' : 'Export redacted contact point'}
+        tooltip={canReadSecrets ? 'Export redacted contact point' : 'Export contact point'}
```

Comment:
The message is flipped, maybe you meant "Export **decrypted** contact point" for the truthy option?

Reply:
I mistook redacted for decrypted... You're right.